io/pci/pci_hotplug.py

```diff
@@ -119,6 +125,18 @@ def is_removed():
 
         return wait.wait_for(is_removed, timeout=10) or False
 
+    def nvme_splitter_slot(self, pci_addrs):
+        """
+        Returs the physical slot of a nvme splitter drive
+
+        :param pci_addrs: pci_adress of the drive
+        :rtype: string
+        """
+        process.run("drmgr -c pci", ignore_status=True, shell=False)
+        cmd = f"lsslot -c pci | grep {pci_addrs}"
```

Comment:
never use pipe to parse the output .. will be difficult to debug the code .. instead use python logic

Reply:
Done